pkgs/development/libraries/dbus/default.nix

```diff
@@ -1,92 +1,115 @@
-{ stdenv, lib, fetchurl, pkgconfig, expat, systemd
-, libX11 ? null, libICE ? null, libSM ? null, x11Support ? (stdenv.isLinux || stdenv.isDarwin) }:
+{ stdenv
+, lib
+, fetchurl
+, pkgconfig
+, expat
+, systemd
+, libX11 ? null
+, libICE ? null
+, libSM ? null
+, x11Support ? (stdenv.isLinux || stdenv.isDarwin)
+, dbus
+}:
 
 assert x11Support -> libX11 != null
```

Comment:
Maybe something like this would better depict the associativity:

```
assert
  x11Support ->
    libX11 != null && libICE != null && libSM != null;
```